src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -370,3 +370,19 @@ class ArchiveConfig(SingletonModel):
 
     class Meta:
         verbose_name = _("archive configuration")
+
+
+class StandardReviewAnswer(models.Model):
+    reason = models.CharField(
+        verbose_name=_("reason"),
+        max_length=200,
+        help_text=_("Reason for the reviewer to request changes to a list."),
+    )
+    order = models.PositiveSmallIntegerField(
```

Comment:
Just replace this with: django-ordered-model and use admin integration: https://github.com/bfirsh/django-ordered-model#admin-integration